src/xnnpack/reshape-helpers.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2023 Google LLC
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+
+#pragma once
+
+#include <xnnpack.h>
+#include <xnnpack/subgraph.h>
+
+enum xnn_status reshape_unary_elementwise_op(
+  const struct xnn_operator_data* opdata,
+  struct xnn_value* values,
+  size_t num_values,
+  size_t old_workspace_size,
+  pthreadpool_t threadpool);
+
+enum xnn_status reshape_binary_elementwise_op(
```

Comment:
nit: can we move this to another PR?

Reply:
Out bot exports a chain of commits as separate PRs, but each exported PR contains the parent commits also. So despite appearing here as new code, this was introduced in https://github.com/google/XNNPACK/pull/5831